docs/platform/terms_and_conditions.md

```diff
@@ -0,0 +1,171 @@
+# Terms and conditions
+
+## Introduction
+
+1. These terms and conditions shall govern your use of our platform, app and data platforms accessible via the website (Analytics and Netmanager) and API, all referred to collectively as our ‘platform’.
+2. By using our platform, you accept these terms and conditions in full; accordingly, if you disagree with these terms and conditions or any part of these terms and conditions, you must not use our platform.
+3. If you register with our platform, submit any material to our platform or use any of our platform services, we will ask you to expressly agree to these terms and conditions.
+4. Our platform uses cookies; by using our platform or agreeing to these terms and conditions, you consent to our use of cookies in accordance with the terms of our Privacy and cookies policy.
+
+## Credit
+
+This document was created using a template from [SEQ Legal](https://seqlegal.com/).
+
```

Comment:
[platform terms and conditions](https://github.com/airqo-platform/AirQo-frontend/pull/956/commits/17c0953e01a846402341eebcda14317135514a78)